homeassistant/components/image_processing/__init__.py

```diff
@@ -38,11 +38,30 @@
 ATTR_FACES = "faces"
 ATTR_GENDER = "gender"
 ATTR_GLASSES = "glasses"
+ATTR_MATCHES = "matches"
 ATTR_MOTION = "motion"
+ATTR_SUMMARY = "summary"
 ATTR_TOTAL_FACES = "total_faces"
+ATTR_TOTAL_MATCHES = "total_matches"
 
-CONF_SOURCE = "source"
+CONF_AREA = "area"
+CONF_AUTH_KEY = "auth_key"
+CONF_BOTTOM = "bottom"
+CONF_CATEGORIES = "categories"
+CONF_CATEGORY = "category"
 CONF_CONFIDENCE = "confidence"
+CONF_COVERS = "covers"
+CONF_DETECTOR = "detector"
+CONF_FILE_OUT = "file_out"
+CONF_GRAPH = "graph"
+CONF_LABELS = "labels"
+CONF_LEFT = "left"
+CONF_MODEL = "model"
+CONF_MODEL_DIR = "model_dir"
+CONF_RIGHT = "right"
+CONF_SOURCE = "source"
+CONF_TOP = "top"
+CONF_URL = "url"
```

Comment:
We should only define common constants for constants that are common between a majority of image processing platforms. This doesn't seem the case for most of these constants.